airframe-rx-html/src/main/scala/wvlet/airframe/rx/html/package.scala

```diff
@@ -59,6 +59,14 @@ package object html {
     def noValue: HtmlNode                             = HtmlAttribute(name, true, namespace)
   }
 
+  class HtmlEventHandlerOf[E](name: String, namespace: Namespace = Namespace.xhtml) {
+    def apply[U](v: E => U): HtmlNode  = HtmlAttribute(name, v, namespace)
+    def ->[U](v: E => U): HtmlNode     = apply(v)
+    def apply[U](v: () => U): HtmlNode = HtmlAttribute(name, v, namespace)
+    def ->[U](v: () => U): HtmlNode    = apply(v)
```

Comment:
- `E => U` is for `e: MouseEvent => ...` 
- `() => U` is for ` () =>. ...` which ignores event